admin-site/src/views/Home.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <HelloWorld />
+  <AdminPortal/>
```

Comment:
Delete the link to <AdminPortal/> and just had some placeholder text for now like "Home Page"